src/sentry/api/endpoints/project_tags.py

```diff
@@ -10,7 +10,7 @@
 
 class ProjectTagsEndpoint(ProjectEndpoint):
     def get(self, request, project):
-        tag_keys = tagstore.get_tag_keys(project.id)
+        tag_keys = sorted(tagstore.get_tag_keys(project.id), key=lambda x: x.key)
```

Comment:
I'm indifferent on whether we force Postgres to do the `ORDER BY` for us. Seemed better to let callers just sort the keys?

Reply:
Yeah, I think this is fine for now. We don’t have a ton of stuff to sort. And less work for the databass the better imo.